editor/src/components/canvas/controls/select-mode/padding-resize-control.tsx

```diff
@@ -164,15 +154,16 @@ const PaddingResizeControlI = React.memo(
         <div
           data-testid={paddingControlHandleTestId(props.edge)}
           onMouseDown={onEdgeMouseDown}
-          onMouseMove={onMouseMove}
           onMouseEnter={hoverStart}
           onMouseLeave={hoverEnd}
           onMouseUp={onMouseUp}
           style={{
+            pointerEvents: 'all',
             visibility: shown ? 'visible' : 'hidden',
             position: 'absolute',
             padding: hitAreaWidth,
             cursor: cursor,
+            zIndex: 1,
```

Comment:
obligatory "does this cause conflicts with any modals we display?" question? FWIW I don't think it would.

Reply:
I checked, as far as I can see it doesn't